tutorial/chapter1.md

```diff
@@ -1,41 +1,219 @@
+# Chapter 1
+
 <p align="right">
-  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+  <img width="25%" height="25%" src="../images/klepsydra_logo.jpg">
 </p>
 
-# Chapter 1
-
-## Table of contents
-1. [Technical requirements](#technical-requirements)
-2. [Introduction](#introduction)
-3. [Core API example](#core-api-example)
+- [Technical requirements](#technical-requirements)
+- [Introduction](#introduction)
+- [Event loop example 1](#event-loop-example-1)
+- [Event loop example 2](#event-loop-example-2)
+- [Image multiplexer example](#image-multiplexer-example)
 
-## Technical requirements <a name="technical-requirements"></a>
+## Technical requirements
 
 Installation of Klepsydra Core, Klepsydra Robotics, Optionally ROS, DDS.
 
 Klepsydra is installed (by default) to /usr/local by CMake. To develop an application using Klepsydra use the cmake command `find_package(Klepsydra)` in the CMakeLists file. With this command, the Klepsydra includes are available in the variable `KLEPSYDRA_INCLUDE_DIRS` and the Klepsydra libraries are available using the variable `KLEPSYDRA_CORE_LIBRARIES`. The CMakeLists file provided for compiling the examples in the tutorial demonstrates how `KLEPSYDRA_CORE_LIBRARIES` variable is used.
```

Comment:
Nitpick:
/usr/local to `/usr/local`
/usr/local/include/klepsydra to `/usr/local/include/klepsydra`